docs/storage-image-processing-api/types/interpolator.mdx

```diff
@@ -27,7 +27,7 @@ by a factor of 2.
 **nohalo**
 
 [Nohalo interpolation](http://eprints.soton.ac.uk/268086/).
-Prevents acutance but typically reduces performance by a factor of 3.
+Prevents acceptance but typically reduces performance by a factor of 3.
```

Comment:
Thanks! All good except this one, the word can be found here for context https://sharp.pixelplumbing.com/api-utility